GnuPlot.php

```diff
@@ -168,8 +171,9 @@ protected function sendInit()
         if ($this->timeFormat) {
             $this->sendCommand('set xdata time');
             $this->sendCommand('set timefmt "'.$this->timeFormat.'"');
-            $this->sendCommand('set format x "'.$this->timeFormat.'"');
-            $this->sendCommand('set xtics rotate by 45 offset -6,-3');
```

Comment:
With the rotation applied, the text was placed outside of the generated png, so it wasn't usable. I guess its the "offset -6,-3" that does it?